package.json

```diff
@@ -128,6 +128,14 @@
     "providers/**/*.{ts,json}": [
       "prettier --ignore-path ./.prettierignore --write",
       "stop-only --file"
+    ],
+    "docs/**/*.{ts,tsx,json}": [
+      "prettier --ignore-path ./.prettierignore --write",
+      "stop-only --file"
+    ],
```

Comment:
A small questions let me know what you think @venarius and @davidsoderberg. We actually have a lint-staged under each package (or mostly under each package) so the individual package will define their own checks depending on the type. We can add a `precommit` script for the individual packages to invoke it's own `lint-staged` command. And from the top we can just use nx to run precommit on all packages. 

What do you think?

Reply:
You can find example for this here: 
Root: https://github.com/scopsy/nestjs-monorepo-starter/blob/master/package.json
Lint Staged on individual package: https://github.com/scopsy/nestjs-monorepo-starter/blob/903cda1938c400e947f60d68595fb22ca7ea2795/apps/api/package.json#L96

Reply:
I personally think stuff like this should be defined once globally for the whole monorepo instead of each package in it having its own version of it

Reply:
This makes sense, so let's leave it like this 🙏